sequencer/src/persistence/fs.rs

```diff
@@ -698,6 +702,41 @@ impl SequencerPersistence for Persistence {
 
         Ok(map)
     }
+
+    async fn load_upgrade_certificate(
+        &self,
+    ) -> anyhow::Result<Option<UpgradeCertificate<SeqTypes>>> {
+        let inner = self.inner.read().await;
+        let path = inner.upgrade_certificate_dir_path();
+        if !path.is_file() {
+            return Ok(None);
+        }
+        let bytes = fs::read(&path).context("read")?;
+        Ok(Some(
+            bincode::deserialize(&bytes).context("deserialize upgrade certificate")?,
+        ))
+    }
+
+    async fn store_upgrade_certificate(
```

Comment:
I think we should add some tests for this:

1. to verify that storing and loading works
2. to verify that we store at most 1 upgrade certificate 
3. to verify that cleanup works

@imabdulbasit can you advise @lukeiannucci on how to add these tests.